examples/native-external-references-mockshop/functions/utils.ts

```diff
@@ -1,7 +1,6 @@
-import { FunctionEventContext } from '@contentful/node-apps-toolkit';
 import { Product } from './types';
 
-export const getMockShopUrl = (context: FunctionEventContext<Record<string, any>>) => {
+export const getMockShopUrl = (context: { appInstallationParameters?: Record<string, any> }) => {
```

Comment:
I would rather use the context type
```suggestion
export const getMockShopUrl = (context: FunctionEventDeliveryContext<InstallationParameters>) => {
```
or refactor the function to 
```suggestion
export const getMockShopUrl = (parameters: InstallationParameters) => {
```